control-plane/pkg/kafka/topic.go

```diff
@@ -31,6 +31,8 @@ const (
 	DefaultTopicNumPartitionConfigMapKey      = "default.topic.partitions"
 	DefaultTopicReplicationFactorConfigMapKey = "default.topic.replication.factor"
 	BootstrapServersConfigMapKey              = "bootstrap.servers"
+
+	TopicAnnotation = "default.topic"
```

Comment:
Can you explain why we need to store the topic in the annotation?

Reply:
https://github.com/knative-sandbox/eventing-kafka-broker/pull/2601#discussion_r963534251